zerver/openapi/zulip.yaml

```diff
@@ -1293,6 +1293,190 @@ paths:
                           ],
                         "upload_space_used": 571946,
                       }
+  /drafts:
+    get:
+      operationId: fetch_drafts
+      tags: ["drafts"]
+      description: |
+        Fetch all drafts for the current user.
+
+        `GET {{ api_url }}/v1/drafts`
```

Comment:
Can you add `**Changes** and API_FEATURE_LEVEL for these new endpoints?

Reply:
Would this be considered a 4.0 feature? I'm not sure about where to exactly put these changes. Also would the `API_FEATURE_LEVEL` become 31 or something else?

Reply:
Yes, exactly, all new features go under the 4.0 heading until we release 4.0 -- and the API_FEATURE_LEVEL just gets incremented by 1 for each feature.